beacon-chain/powchain/block_reader.go

```diff
@@ -72,8 +80,9 @@ func (s *Service) BlockTimeByHeight(ctx context.Context, height *big.Int) (uint6
 // This is a naive implementation that will use O(ETH1_FOLLOW_DISTANCE) calls to cache
 // or ETH1. This is called for multiple times but only changes every
 // SlotsPerEth1VotingPeriod (1024 slots) so the whole method should be cached.
+// How should caching the whole method be done?
```

Comment:
? do we need this comment

Reply:
Just for personal tracking as I try to figure out whats needed, will remove before the PR is ready for merging